src/frontend/js/components/users.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes, Component } from 'react';
+
+import UserEntry from './user-entry';
+
+export default class Users extends Component {
+  static propTypes: {
+    users: PropTypes.object.isRequired,
+    fetchUsers: PropTypes.func.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.fetchUsers = this.fetchUsers.bind(this);
+    this.fetchUsers();
+  };
+
+  fetchUsers() { this.props.fetchUsers() };
+
+  render() {
+    return (
+      <div className="app">
+        <div className="list x">
+          {this.props.users.map(user => (
+            <UserEntry key={user.get('id')} user={user} />
```

Comment:
It might be due to how small this app is, but without visible code reuse, putting UserEntry in a separate component is a bit on the overengineering side.

Totally different thing if you would reuse UserEntry in user.jsx